src/components/signoff/SimpleReview/index.tsx

```diff
@@ -102,9 +102,20 @@ export default function SimpleReview({
     // get collection data
     async function getRecords() {
       if (destCid && destBid && sourceBid && sourceCid) {
-        const newRecords = await fetchRecords(sourceBid, sourceCid);
-        const oldRecords = await fetchRecords(destBid, destCid);
-        setRecords({ oldRecords, newRecords, loading: false });
+        try {
+          const newRecords = await fetchRecords(sourceBid, sourceCid);
+          const oldRecords = await fetchRecords(destBid, destCid);
+          setRecords({ oldRecords, newRecords, loading: false });
+        } catch (ex) {
+          if (ex.data?.code === 401) {
+            setRecords({
+              ...records,
+              loading: false,
+            });
+          } else {
+            console.error(ex);
```

Comment:
Can we show an error notification?

Reply:
That's a good idea, added notification and updated tests.